src/components/Dataset/DatasetReady.js

```diff
@@ -17,14 +17,23 @@ export const DatasetReady = ({ dataset }) => {
   const { downloadDataset } = useDatasetManager()
   const { setResponsive } = useResponsive()
   const { validateToken } = useToken()
-  const hasToken = validateToken()
-  const [termsOfUse, setTermsOfUse] = useState(hasToken)
+  const [acceptedTerms, setAcceptedTerms] = useState(false)
```

Comment:
I don't think this should be defined local to the component. This should be in `useToken`. It should start false before being queried, then updated when validated.

Reply:
We're removing `useToken` and moving user token validation to the refinebio context during application load, rather than at the time of file downloads (#413). As a result, these checks are no longer necessary and will be removed from the components including this one.